src/PrestaShopBundle/Bridge/AdminController/AdminControllerTrait.php

```diff
@@ -88,7 +56,7 @@ public function getHelperListBridge(): HelperListBridge
     public function addAction(ActionInterface $action): void
```

Comment:
Since you started implementing the bridge in the PR(good call by the way) maybe this method should also be part of the bridge controller object instead? Or in the `ControllerConfiguration` itself? (or maybe both and the bridge is just a proxy to the configuration method).

This way the trait on the controller has less methods and less risks of conflicting method names, however it can still easily add actions using `$this->controllerCOnfiguration->addAction()` or `$this->legacyControllerBridge->addAction()`